extensions/import/src/wizard/flatFileWizard.ts

```diff
@@ -49,4 +77,19 @@ export function flatFileWizard(provider: FlatFileProvider) {
 
 		wizard.pages = [page1, page2, page3, page4];
 		wizard.open();
-}
\ No newline at end of file
+}
+
+function importDataStatus() : { promise: Promise<boolean>,
+						 resolve: (value: boolean | PromiseLike<boolean>) => void
+						 reject: (reason?: any) => void
+						} {
+	let outResolve, outReject;
+	return {
+		promise: new Promise<boolean>((resolve, reject) => {
+			outResolve = resolve;
+			outReject = reject;
+		}),
```

Comment:
This formatting looks weird. From the command line you can install our formatting tool `tsfmt` by running `npm install -g tsfmt`, and then use it to format the file by running `tsfmt -r <filename>` and then committing the new version of the file